lib/ssl/src/ssl.erl

```diff
@@ -1215,13 +1247,17 @@ rsa
 ]
 ```
 
-Support for \{md5, rsa\} was removed from the the TLS-1.2 default in ssl-8.0
-(OTP 22) and support for SHA1 \{sha, _\} and SHA224 \{sha224, _\} was removed in
-ssl-11.0 (OTP 26)
+> #### Change {: .info }
+>
+> - Support for `{md5, rsa}` was removed from the TLS-1.2 default in
+>   ssl-8.0 (Erlang/OTP 22).
+> - Support for `{sha, _}` (SHA1) and `{sha224, _}` (SHA224) was removed
```

Comment:
(SSH224) seems superfluous, and saying it is a small part of SHA2 is maybe not relevant here.  But I think  it can be relevant to keep (SHA1)  as it default is referred to as SHA only for legacy reasons.

Reply:
OK. I've removed the "(SHA224)" part.